#Action message for requesting a collision free plan of a linear motion

#Space of planning, can be either 'joint' or 'tool'
string planning_space

#Starting joint configuration, if empty the motion starting point is the current point
sensor_msgs/JointState initial_joint_config

#waypoints of the trajectory. A list that specifies waypoints of the trajectory. The last point #corresponds to the motion's target. Can be either joint confs, poses or named targets

sensor_msgs/JointState[]    jointWaypoints
geometry_msgs/PoseStamped[] pose_waypoints

#Constraints of the motion
moveit_msgs/Constraints motion_constrains
---
moveit_msgs/MotionPlanResponse linear_motion_response
bool success
---
string message
